src/core_atmosphere/physics/mpas_atmphys_driver_convection.F

```diff
@@ -112,6 +112,8 @@ subroutine allocate_convection(configs)
 
  call mpas_pool_get_config(configs,'config_convection_scheme',convection_scheme)
 
+!!!write(*,*) "*** allocate 0"
+
```

Comment:
@cponder I think we need to get rid of all these debug messages by submitting one more commit

Reply:
There are multiple debug messages below. the above comment applies to all of them